app/controllers/dashboard_controller.rb

```diff
@@ -56,7 +56,8 @@ def app_config
       FB_APP_ID: GlobalConfigService.load('FB_APP_ID', ''),
       FACEBOOK_API_VERSION: 'v14.0',
       IS_ENTERPRISE: ChatwootApp.enterprise?,
-      AZURE_APP_ID: ENV.fetch('AZURE_APP_ID', '')
+      AZURE_APP_ID: ENV.fetch('AZURE_APP_ID', ''),
+      GOOGLE_OAUTH_ENABLED: ENV.fetch('GOOGLE_OAUTH_ENABLED', nil)
```

Comment:
I don't think this is needed. Let us check for the presence of GOOGLE_OAUTH_CLIENT_ID.